test/common/dns.js

```diff
@@ -0,0 +1,290 @@
+'use strict';
+
+// Naïve DNS parser/serializer.
+
+require('../common');
```

Comment:
@Trott Sorry for being so late, I’ve updated this PR with `eslint-disable required-modules`